client/elements/sc-site-layout.js

```diff
@@ -308,50 +432,67 @@ class SCSiteLayout extends LitLocalized(LitElement) {
       this.shadowRoot.querySelector('#sutta-info').hide();
     });
     let rootDOM = this.shadowRoot;
-    addEventListener('scroll', throttle(500, () => {
-      let transitionStyle = 'transform 200ms ease-in-out';
-      rootDOM.getElementById('universal_toolbar').style.transition = transitionStyle;
-      rootDOM.getElementById('breadCrumb').style.transition = transitionStyle;
-      rootDOM.getElementById('mainTitle').style.transition = transitionStyle;
-      rootDOM.getElementById('subTitle').style.transition = 'transform 300ms ease-in-out';
-
-      if (this.changedRoute.path === '/' && (document.body.scrollTop > 100 || document.documentElement.scrollTop > 100)) {
-        rootDOM.getElementById('universal_toolbar').style.transform = 'translateY(-120px)';
-        rootDOM.getElementById('breadCrumb').style.transform = 'translateY(120px)';
-        rootDOM.getElementById('mainTitle').style.transform = 'translateY(74px) scale(0.667)';
-        rootDOM.getElementById('subTitle').style.opacity = '0';
-        rootDOM.getElementById('subTitle').style.transform = 'scale(0)';
-        if (window.innerWidth < 480) {
-          rootDOM.getElementById('mainTitle').style.transform = 'translateY(70px) scale(0.667)';
+    addEventListener(
+      'scroll',
+      throttle(500, () => {
+        let transitionStyle = 'transform 200ms ease-in-out';
```

Comment:
nit: it can be constant

Reply:
lets leave it for now, I only reformatted those components for now, lets not touch this.

Reply:
I will gradually modify these codes according to eslint's prompts :)